### PR TITLE
Fix IoT Models Repository Readme issues

### LINKED ISSUE #20215: Azure IoT Models Repository Readme issues
1.
Section [link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/modelsrepository/Azure.IoT.ModelsRepository#azure-iot-models-repository-client-library-for-net):
image
Suggestion: Add hyperlink for  `Package (nuget)`  https://www.nuget.org/packages/Azure.IoT.ModelsRepository  

@jongio for notification.

### PR DESCRIPTION
**Pull request by v-yilinhu**
Fix https://github.com/Azure/azure-sdk-for-net/issues/20215

@jongio for notification.

### CODE REVIEW

**Comment by v-xuto** [comment]
@jongio Who should we ping to review this PR?

**Review by drwill-ms — APPROVED** [review]

**Comment by digimaun** [comment]
It would probably help to change the title. There are no IoT Models Repository changes in this PR....and there is a label of `Storage` associated.

**Comment by v-xuto** [comment]
@drwill-ms - Could you help us merge this PR?

File: sdk/quantum/Azure.Quantum.Jobs/README.md [hunk 1]
```diff
@@ -180,15 +180,17 @@ testing, and contributing to this library.
 This project welcomes contributions and suggestions.  Most contributions require
 you to agree to a Contributor License Agreement (CLA) declaring that you have
 the right to, and actually do, grant us the rights to use your contribution. For
-details, visit [cla.microsoft.com][cla].
+details, visit https://cla.microsoft.com.
```
Comment (drwill-ms) [inline sdk/quantum/Azure.Quantum.Jobs/README.md:183]:
```suggestion
details, visit <https://cla.microsoft.com>.
```

Comment (drwill-ms) [reply sdk/quantum/Azure.Quantum.Jobs/README.md:183]: https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

**Comment by drwill-ms** [comment]
> @drwill-ms - Could you help us merge this PR?

I think we should see sign-off from the owners of the other two files before we merge.

**Comment by v-xuto** [comment]
@drwill-ms  We have updated and removed the other two files. Please re-review.

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**